index.html

```diff
@@ -36,6 +36,131 @@ <h1 class="heading-1">
+        <span>See This Project</span>
+        <span><ion-icon name="arrow-forward-outline"></ion-icon></span> 
+        </button>
+        </div>
+        <div class="background-color">
+            <img src="img/Rectangle24.svg" alt="Rectangle" class="projects-img">
+            <h3 class="projects-heading-2">Project Name goes here</h3>
+            <ul class="projects-card">
+            <li class="project-list"><button class="list-btn" type="button">Ruby On Rauls</button></li>
+            <li class="project-list"><button class="list-btn" type="button">Ruby On Rauls</button></li>
+            <li class="project-list"><button class="list-btn" type="button">Javascript</button></li>
+        </ul>
+        <button class="project-button" type="button">
+        <span>See This Project</span>
+        <span><ion-icon name="arrow-forward-outline"></ion-icon></span> 
+        </button>
+        </div>
+        <div class="background-color">
+            <img src="img/Rectangle25.svg" alt="Rectangle" class="projects-img">
+            <h3 class="projects-heading-2">Project Name goes here</h3>
+            <ul class="projects-card">
+            <li class="project-list"><button class="list-btn" type="button">Ruby On Rauls</button></li>
+            <li class="project-list"><button class="list-btn" type="button">Ruby On Rauls</button></li>
+            <li class="project-list"><button class="list-btn" type="button">Javascript</button></li>
+        </ul>
+        <button class="project-button" type="button">
+        <span>See This Project</span>
+        <span><ion-icon name="arrow-forward-outline"></ion-icon></span> 
+        </button>
+        </div>
+        <div class="background-color">
+            <img src="img/Rectangle26.png" alt="Rectangle" class="projects-img">
+            <h3 class="projects-heading-2">Project Name goes here</h3>
+            <ul class="projects-card">
+            <li class="project-list"><button class="list-btn" type="button">Ruby On Rauls</button></li>
+            <li class="project-list"><button class="list-btn" type="button">Ruby On Rauls</button></li>
+            <li class="project-list"><button class="list-btn" type="button">Javascript</button></li>
+        </ul>
+        <button class="project-button" type="button" >
+        <span>See This Project</span>
+        <span><ion-icon name="arrow-forward-outline"></ion-icon></span> 
+        </button>
+        </div>   
+    </section>
+    <section class="about">
+        <h1 class="heading-1">
+            About
+        </h1>
+        <p class="about-par">
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod
+        </p>
+        <button class="cv-btn" type="button">
+        <span>Get My CV</span>
+        <span><ion-icon name="download-outline"></ion-icon></span>
```

Comment:
- [ ] The button (e.g., "Get My CV") is a link a or button tag, and follow the button styling: Active, Hover and Pressed (see design), kindly make see this project as a link or button with a hover color and pressed.